Examples/Google/SpeechToText/Sources/SpeechService.swift

```diff
@@ -16,6 +16,7 @@
 
 import GRPC
 import Logging
+import Foundation
```

Comment:
The formatter doesn't like the order of these imports, I don't think we need `Foundation` here so let's just remove it.

```suggestion
```

Reply:
When I remove the import, the compiler has trouble finding `Data` for the `stream(_ data: Data, ..)` method below. Not 100% certain, my hunch would be that when removing Cocoapods a project setting changed that allowed it to not be required. 

I've corrected the order of the imports 👍

Reply:
Ah, great, thanks!